src/gui/layout/qgslayoutelevationprofilewidget.cpp

```diff
@@ -674,6 +716,12 @@ void QgsLayoutElevationProfileWidget::setGuiElementValues()
   mSpinMinElevation->setValue( mProfile->plot()->yMinimum() );
   mSpinMaxElevation->setValue( mProfile->plot()->yMaximum() );
 
+  mSubsectionsActivateCheck->setChecked( mProfile->subsectionsSymbol() != nullptr );
```

Comment:
```suggestion
  mSubsectionsActivateCheck->setChecked( mProfile->subsectionsSymbol() );
```

(see https://github.com/qgis/QGIS-Enhancement-Proposals/blob/master/qep-314-coding-style.md#3-language-features, 3.3)

Reply:
Done.